ca/ca.go

```diff
@@ -79,6 +79,46 @@ type certProfilesMaps struct {
 	profileByName map[string]*certProfileWithID
 }
 
+// caMetrics holds various metrics which are shared between caImpl, ocspImpl,
+// and crlImpl.
+type caMetrics struct {
```

Comment:
Is this refactoring meant to be part of the "fix a reference bug" PR? It's not mentioned in the description.

Reply:
Yeah I guess it's buried at the end in "and unify all three implementations into a single shared type". I'll update the PR description to place more weight on this refactoring.